### PR TITLE
Update README.md with swarm mode usage

### PR DESCRIPTION
**Pull request by ishantt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ By default, the ipam-plugin assumes that the "Cloud Network Automation" licensed
 
 Plugin is installed by pulling the infoblox/ipam-plugin from the docker store and setting its environment variables.
 
+In Docker swarm mode the plugin needs to be installed on all the nodes.
```
Comment (yuewko) [inline README.md:51]: You might want to make it clear that the following are "options" (i.e. not two "steps")

File: README.md [hunk 2]
```diff
@@ -172,7 +174,113 @@ $ docker run -it --network priv-net alpine /bin/sh
     inet6 fe80::42:aeff:feaa:e41c/64 scope link
        valid_lft forever preferred_lft forever
 / #
+```
+### Using Plugin in Swarm mode
 
+Currently the plugin supports the MACVLAN network driver in the swarm mode.
+The plugin needs to be installed in all the swarm nodes.
```
Comment (yuewko) [inline README.md]: I suggest saying something like: "Before performing the following steps, the plugin needs to be installed....."

**Review by yuewko — COMMENTED** [review]
Very good. Just a couple of nits..

**Comment by ishantt** [comment]
Thanks Ko! I will fix these.

**Review by yuewko — APPROVED** [review]
LGTM!

**Pull request merged**